### DISCUSSION TITLE
Air gapped environments?

### DISCUSSION THREAD

**Discussion by USAF-cbentkowski** [DependencyTrack/dependency-track#2639] [General]
We're looking at using Dependency Track, but our environment is in an air gapped environment. Is there a way to manually load the feeds that Dependency Track requires from the Internet via files in an air gapped environment?

**Comment by syalioune** [comment]
Well it depend on the source. The ones that can be changed are in table `CONFIGPROPERTY` (some can be changed using the UI)

![image](https://user-images.githubusercontent.com/6144741/229605497-1722e823-2af7-4b79-b0a0-3156474a0205.png)

Though `Snyk` can't be mirrored as it rely on (paid) proprietary data fetchable using adhoc queries. Same goes for `OSS Index` whose URL is not configurable anyway.

`Github Advisory` is periodically mirrored by Dependency Track using Github GraphQL API but the URL is not configurable.
